docs/desi/desi.rst

```diff
@@ -0,0 +1,97 @@
+.. _astroquery.desi:
+
+*********************************************
+DESI LegacySurvey Queries (`astroquery.desi`)
+*********************************************
+
+Getting started
+===============
+
+This module provides a way to query the DesiLegacySurvey service.
+Presented below are examples that illustrate the different types of queries
+that can be formulated.
+
+Query a region
+==============
+
+This example shows how to query a certain region with DesiLegacySurvey.
+We'll use a set of coordinates that define the region of interest,
+and search within a 5 arcmin radius.
+
+.. doctest-remote-data::
+
+    >>> from astroquery.desi import DESILegacySurvey
+    >>> from astropy.coordinates import Angle, SkyCoord
+    >>> ra = Angle('11h04m27s', unit='hourangle').degree
+    >>> dec = Angle('+38d12m32s', unit='hourangle').degree
+    >>> coordinates = SkyCoord(ra, dec, unit='degree')
+    >>> radius = Angle(5, unit='arcmin')
+    >>> table_out = DESILegacySurvey.query_region(coordinates, radius, data_release=9)
+    >>> print(table_out[:5])
+         ls_id              dec                ra         ... type wise_coadd_id
+                                                          ...
+    ---------------- ----------------- ------------------ ... ---- -------------
+    9907734382838387 38.12628495570797  166.0838654387131 ...  PSF      1667p378
+    9907734382838488 38.12798336424771  166.0922968862182 ...  PSF      1667p378
+    9907734382838554 38.12858283671958  166.0980673954384 ...  REX      1667p378
+    9907734382838564 38.12840803351445  166.09921863682337 ...  EXP      1667p378
+    9907734382838584 38.12836885301038  166.10070750146636 ...  REX      1667p378
+
+The result is an astropy.Table.
+
+Get images
+==========
+
+To download images for a certain region of interest,
+we can define our region in the same way used in the example above.
+
+.. doctest-remote-data::
+
+    >>> from astroquery.desi import DESILegacySurvey
+    >>> from astropy.coordinates import Angle, SkyCoord
+    >>>
+    >>> ra = Angle('11h04m27s', unit='hourangle').degree
+    >>> dec = Angle('+38d12m32s', unit='hourangle').degree
+    >>> radius_input = 0.5
+    >>> pixels = 60
+    >>>
+    >>> pos = SkyCoord(ra, dec, unit='degree')
+    >>> radius = Angle(radius_input, unit='arcmin')
+    >>> im = DESILegacySurvey.get_images(pos, pixels, radius, data_release=9)
+
+    >>> # All the information we need can be found within the object "im".
+
```

Comment:
Not certain, but I think the whitespace here prevented the variable from being carried along to the next section, resulting in the current build failure.
```suggestion
    >>>
    >>> # All the information we need can be found within the object "im".
    >>>
```